### PR TITLE
Setup mac

### PR DESCRIPTION
**Pull request by terezaif**

### CODE REVIEW

File: mac-setup-with-just/setup-python.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#setup python
+echo -e "JUST SETTING UP PYTHON"
+just setup-python
+
+#install python version using pyenv
+echo "INSTALLING PYENV 3.11.4"
+pyenv install 3.11.4
+
+
+#setup ssh connection to github
```
Comment (neinkeinkaffee) [inline mac-setup-with-just/setup-python.sh]: Maybe this should not be run automatically as part of the standard setup. It generates an SSH key with the default name `id_rsa`. If people already have a key of that name, they might run into the risk of overwriting it.

File: mac-setup-with-just/setup-just-and-terminal.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#install just
+echo -e "BREWING JUST"
+brew install just
+
+#configuring your terminal part 1
```
Comment (neinkeinkaffee) [inline mac-setup-with-just/setup-just-and-terminal.sh]: My understanding was that we want to take this and `setup-terminal-part2` out of the standard run, as it's changing the terminal configuration - or is this only installing things and not actually doing the switch?

File: mac-setup-with-just/setup-python.sh (hunk 1 above)
Comment (neinkeinkaffee) [reply mac-setup-with-just/setup-python.sh]: So my suggestion would be that we take the `just setup-terminal-part1`, `just setup-terminal-part2` and `just setup-ssh` out of this bash script, and add a line to the README instead that tells people to run them if they want to configure their terminal and their first SSH key (if they don't have one).

File: mac-setup-with-just/setup-just-and-terminal.sh (hunk 2 above)
Comment (terezaif) [reply mac-setup-with-just/setup-just-and-terminal.sh]: done

File: mac-setup-with-just/setup-python.sh (hunk 1 above)
Comment (terezaif) [reply mac-setup-with-just/setup-python.sh]: done!

**Review by DeboraViertel23 — APPROVED** [review]

**Pull request merged**